lib/prompts/getSystemPrompt.ts

```diff
@@ -18,7 +18,10 @@ export async function getSystemPrompt({
 }): Promise<string> {
   const resolvedArtistId = artistId || (await getArtistIdForRoom(roomId || ""));
 
-  let systemPrompt = `${SYSTEM_PROMPT} 
+  let systemPrompt = `
+  Current date and time in UTC: ${new Date().toISOString()}
+  
+  ${SYSTEM_PROMPT} 
```

Comment:
grouping - keep all `dynamic` data together in the system prompt for easier maintainability.
```suggestion
  let systemPrompt = `${SYSTEM_PROMPT} 
  Current date and time in UTC: ${new Date().toISOString()}
```